docs/error-reporting/platform-integrations/real-devices/setup.md

```diff
@@ -0,0 +1,128 @@
+---
+id: setup
+title: Setting Up Backtrace for your Real Device Tests
+sidebar_label: Setup
+description: Link your Sauce Labs real-device tests with Backtrace to view crash and error reports for faster debugging.
+---
+Link your Sauce Labs real-device tests with Backtrace to view crash and error reports for faster debugging. Detecting errors and crashes and resolving them early on is key to the success of your app.
+
+After you’ve completed the steps on this page, crashes and errors associated with your real-device tests (automated or live) will be available for you in Sauce Labs.
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+## Supported Platforms
+|Supported Platforms|Supported Systems|Supported Features|
+|---------|---------|---------|
+|Real devices|Android|Crashes and Errors|
+|Real devices|iOS|Errors|
+
+:::note
+Crashes for iOS apps are currently not supported.
```

Comment:
Looking at this, I think we should add to it :
Crashes for iOS apps are currently not supported when running real device tests on Sauce Labs with Backtrace